From ae4ed3619099ce6dfeb8df4633ace7e7393f8d3a Mon Sep 17 00:00:00 2001
From: Wenping Zhang <wenping.zhang@rock-chips.com>
Date: Fri, 13 Oct 2017 10:56:48 +0800
Subject: [PATCH] ARM: dts: rockchip: improve bus speed of eMMC for rk3229-gva.

add mmc hs200/ddr mode support and increase the max clock frequence.

Change-Id: I7e81dfaf364d88a3e8bf0278a8f771a8179c6627
Signed-off-by: Wenping Zhang <wenping.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-gva-sdk.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/rk3229-gva-sdk.dts b/arch/arm/boot/dts/rk3229-gva-sdk.dts
index 87788335c1ce..7f49a7350a49 100644
--- a/arch/arm/boot/dts/rk3229-gva-sdk.dts
+++ b/arch/arm/boot/dts/rk3229-gva-sdk.dts
@@ -131,6 +131,12 @@
 };
 
 &emmc {
+	max-frequency = <150000000>;
+	cap-mmc-highspeed;
+	mmc-ddr-1_8v;
+	mmc-hs200-1_8v;
+	non-removable;
+	supports-emmc;
 	status = "okay";
 };
 
-- 
2.35.3

